server/src/main/java/com/hedera/block/server/persistence/storage/read/BlockAsDirReader.java

```diff
@@ -57,12 +57,10 @@ class BlockAsDirReader implements BlockReader<Block> {
      * given parameters.
      *
      * @param config the configuration to retrieve the block node root path
-     * @param filePerms the file permissions to set on the block node root path
+     * @param filePerms the file permissions to set on the block node root path, default  will be used if null provided
      */
     BlockAsDirReader(
-            @NonNull final PersistenceStorageConfig config,
-            @NonNull final FileAttribute<Set<PosixFilePermission>> filePerms) {
-
+            @NonNull final PersistenceStorageConfig config, final FileAttribute<Set<PosixFilePermission>> filePerms) {
```

Comment:
I forgot to mention that in the threads, was thinking about that today as well, I was thinking about just `permissions`, but `folderPermissions` is more descriptive. Since #271 will be first, I will change the names to your suggestions when I resolve the merge conflicts and resolve your other comment about the tests from #271.

Reply:
✅ Done.